src/frontend/packages/lib_components/src/common/SchedulingFields/index.tsx

```diff
@@ -219,43 +211,33 @@ export const SchedulingFields = ({
         margin={margin}
         gap="small"
       >
-        <Box>
-          <FormField
-            label={intl.formatMessage(messages.startingAtDateTextLabel)}
-            htmlFor="starting_at_date"
-            margin="none"
-            background={startingAtError ? 'status-error-off' : 'white'}
-            // Fix of the calendar icon still clickable when component disabled (see below)
-            style={{ pointerEvents: disabled ? 'none' : undefined }}
-            height="80%"
-            disabled={disabled}
-          >
-            <DateInput
-              dropProps={{
-                align: { top: 'bottom', left: 'left' },
-                style: {
-                  borderRadius: '4px',
-                  boxShadow: 'rgb(0 0 0 / 23%) 4px 5px 17px',
-                },
-              }}
-              id="starting_at_date"
-              format={intl.locale === 'fr' ? 'dd/mm/yyyy' : 'yyyy/mm/dd'}
-              value={currentStartingAtDate || undefined}
-              onChange={onStartingAtDateInputChange}
-              calendarProps={{
-                bounds: [
-                  DateTime.local().toISO() as string,
-                  DateTime.local().plus({ years: 1 }).toISO() as string,
-                ],
-              }}
-              // TODO : calendar icon still clickable even when component is disabled
-              // need to open an issue on grommet's github
+        <Box data-testid="starting-at-date-picker">
+          <CunninghamProvider>
+            <DatePicker
               disabled={disabled}
+              fullWidth
+              label={intl.formatMessage(messages.startingAtDateTextLabel)}
+              locale={intl.locale}
+              maxValue={DateTime.local().plus({ years: 1 }).toISO() as string}
```

Comment:
I don't remember why we choose to block to 1 year but IMO there is no reason to do that. You can remove it.

Reply:
https://github.com/openfun/marsha/pull/2359/commits/5f743a5a20511d9a266a617f4785c178abae3aa5